examples/openshift/template.yaml

```diff
@@ -0,0 +1,236 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: jitsi
+objects: (( merge ))
+parameters:
+#
+# Basic configuration options
+#
+- name: DOMAIN
+  required: true
+- name: TZ
+  displayName: Timezone
+  required: true
+- name: LOG_LEVEL
+
+
+#
+# HTTPS configuration
+#
+- name: DISABLE_HTTPS
+- name: ENABLE_HTTP_REDIRECT
+
+
+#
+# Authentication configuration
+#
+- name: ENABLE_AUTH
+- name: AUTH_TYPE
+- name: ENABLE_GUESTS
+
+- name: JWT_APP_ID
+- name: JWT_APP_SECRET
+- name: JWT_ACCEPTED_ISSUERS
+- name: JWT_ACCEPTED_AUDIENCES
+- name: JWT_ASAP_KEYSERVER
+- name: JWT_ALLOW_EMPTY
+- name: JWT_AUTH_TYPE
+- name: JWT_TOKEN_AUTH_MODULE
+
+- name: LDAP_URL
+- name: LDAP_BASE
+- name: LDAP_BINDDN
+- name: LDAP_BINDPW
+- name: LDAP_FILTER
+- name: LDAP_AUTH_METHOD
+- name: LDAP_VERSION
+- name: LDAP_USE_TLS
+- name: LDAP_TLS_CIPHERS
+- name: LDAP_TLS_CHECK_PEER
+- name: LDAP_TLS_CACERT_FILE
+- name: LDAP_TLS_CACERT_DIR
+- name: LDAP_START_TLS
+
+
+#
+# A comma separated list of APIs to enable when the JVB is started. The default is none.
+# See https://github.com/jitsi/jitsi-videobridge/blob/master/doc/rest.md for more information
+#
+- name: JVB_ENABLE_APIS
+  required: true
+
+
+#
+# Advanced configuration options
+#
+- name: JVB_PORT
+  required: true
+- name: JVB_TCP_HARVESTER_DISABLED
+  required: true
+- name: JVB_TCP_PORT
+  required: true
+- name: JVB_BREWERY_MUC
+  required: true
+- name: JVB_AUTH_USER
+  required: true
+- name: JVB_AUTH_PASSWORD
+  required: true
+- name: JVB_STUN_SERVERS
+  required: true
```

Comment:
I would suggest settings this to required: false as I was having issues with it when running this template in a openshift cluster which does not have access to the internet, the pods were unable to reach the stun server and as such refused to start up, had to disable this in openshift as it is currently not possible to do it via the .env-file

Reply:
I removed this line. If you think this is everything which has to be done, feel free to resolve this review conversation.